entrypoint.sh

```diff
@@ -11,6 +11,69 @@ readonly CLOUD_PASS=$3
 # For local testing run: ./entrypoint.sh .cluster.dev/minikube-one.yaml AWSUSER AWSPASS
 #
 
+#######################################
+# Run command into wrapper, that print command output only when:
+# - error happened
+# - VERBOSE_LVL=DEBUG
+# Globals:
+#   None
+# Arguments:
+#   command - command that should be executed inside wrapper
+#   bash_opts - additional shell options
+#   fail_on_err - interpret not 0 exit code as error or not.
+#                 Boolean. By default - true.
+#   enable_log_timeout - Print all logs for command after timeout.
+#                        Useful for non-DEBUG log levels.
+#                        By default - 300 seconds (5 min)
+# Outputs:
+#   Writes progress status
+#######################################
+function run_cmd {
+    local command="$1"
+    local bash_opts="${2-""}"
+    local fail_on_err="${3-true}"
+    local enable_log_timeout="${3-300}" # By default - 300 seconds (5 min)
+
+    local bash="/usr/bin/env bash ${bash_opts}"
+
+    # STDERR prints by default
+    # Log STDOUT and continue
+    if [ "$LOG_LVL" == "DEBUG" ]; then
+        DEBUG "Output from '$command'" "" "" 1
+        ${bash} -x -c "$command"
+        exit_code=$?
+
+        [ ${exit_code} != 0 ] && [[ $fail_on_err ]] && ERROR "Execution of '$command' failed" "" "" 1
+        return
+    fi
+
+    # shellcheck disable=SC2091
+    $(${bash} -x -c "$command" >/tmp/log) &
+    proc_pid=$!
+
+    # Print logs in non-debug
+    seconds=0
+    until [ ! -d /proc/$proc_pid ]; do
+        if [[ $seconds -gt $enable_log_timeout ]]; then
+            WARNING "Command '$command' executing more than ${seconds}s. \
+Start print out exist and future output for this command." "%DATE - %MESSAGE" "+%T" 1
+            tail -f -n +0 /tmp/log &
+            tail_pid=$!
+            break
+        fi
+        sleep 1
+        ((seconds++))
+    done
+
+    wait $proc_pid
+    exit_code=$?
+    # `> /dev/null 2>&1 &` and `wait` need to suppress "Terminated' message
+    kill $tail_pid > /dev/null 2>&1 &
```

Comment:
is variable $tail_pid have correct value outside "until" loop? 
(https://stackoverflow.com/questions/16854280/a-variable-modified-inside-a-while-loop-is-not-remembered)

Reply:
Yes. It always try to terminate PID or "" and when I tesst before suppress stdout - it always terminate `tail`

Just recheck and all seems good

```bash
$ export VERBOSE_LVL=NOTICE && rm -rf terraform/aws/backend/.terraform && bash cmd.sh
+ cd terraform/aws/backend
+ terraform init
13:22:18 - Command 'cd terraform/aws/backend && terraform init' executing more than 4s. Start print out exist and future output for this command.
tail_pid inside loop: 4088

Initializing the backend...

Initializing provider plugins...
- Checking for available provider plugins...
- Downloading plugin for provider "aws" (hashicorp/aws) 2.52.0...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
tail_pid outside loop: 4088
```